examples/custom-shared-image.json

```diff
@@ -0,0 +1,50 @@
+{
+    "apiVersion": "vlabs",
```

Comment:
can you please add some docs for the feature? 

- apimodel definition should go in https://github.com/Azure/aks-engine/blob/master/docs/topics/clusterdefinitions.md#L513
- feature documentation can go in:
https://github.com/Azure/aks-engine/blob/master/docs/topics/features.md